src/components/foundation/Spacing.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Box, Flex } from 'grid-styled';
+import theme from '../../theme';
+import Text from '../core/Text';
+import Card from '../core/Card';
+
+const SpacingExample = styled(Card)`
+	padding: 0;
+	height: ${props => props.space}px;
+	margin: 0;
+	font-size: ${props => props.fontSize};
+	font-family: ${props => props.theme.fontFamily.base};
+	background: ${props => props.theme.color.light};
+	border: none;
+`;
+const SpaceSample = ({ space, children }) => <Box m={0}>
```

Comment:
I'd add some space between samples, otherwise it looks like 0 is a label for a 4px bar.